data/geany.glade

```diff
@@ -3731,6 +3731,22 @@
                                     <property name="position">4</property>
                                   </packing>
                                 </child>
+                                <child>
+                                  <object class="GtkCheckButton" id="check_autoclose_btick">
+                                    <property name="label" translatable="yes">Backticks ` `</property>
+                                    <property name="visible">True</property>
+                                    <property name="can_focus">True</property>
+                                    <property name="receives_default">False</property>
+                                    <property name="tooltip_text" translatable="yes">Auto-close backtick when typing an opening one</property>
+                                    <property name="use_underline">True</property>
+                                    <property name="draw_indicator">True</property>
+                                  </object>
+                                  <packing>
+                                    <property name="expand">True</property>
+                                    <property name="fill">True</property>
+                                    <property name="position">3</property>
```

Comment:
Did Glade put the `3` here or did you edit it by hand?

Reply:
@elextr , by hand. I see the position is duplicated. Which pos would you like?

Reply:
Better to do it with Glade now we can use the one from the distro.

As for the order of the items, I don't think it matters, I would just stick it on the end, but let Glade do it.

Reply:
I've never used glade before, but I'll check the Geany docs for instructions.

Is there a Geany IRC channel anywhere that you might be on?

Reply:
Pretty effin lame, dude 
![image](https://user-images.githubusercontent.com/16764864/145699380-c9787d6f-cf27-441d-bc12-561182e61aca.png)

I did a copy and paste, but it won't line up. And I've compared the settings with the others and can't find a difference.

Using Glade 3.38.2 because that's what in the Manjaro repo.

Ah.. I suppose it just isn't compatible... Now I see all these errors when I try to save

```
[properties_dialog > dialog-vbox4 > vbox7 > table5 > file_perm_other_x_check]
	Property 'Focus on click' of object class 'Widget' was introduced in gtk+ 3.20
[properties_dialog > dialog-vbox4 > vbox7 > table5 > file_perm_other_w_check]
	Property 'Focus on click' of object class 'Widget' was introduced in gtk+ 3.20
[properties_dialog > dialog-vbox4 > vbox7 > table5 > file_perm_other_r_check]
	Property 'Focus on click' of object class 'Widget' was introduced in gtk+ 3.20
[properties_dialog > dialog-vbox4 > vbox7 > table5 > file_perm_group_x_check]
	Property 'Focus on click' of object class 'Widget' was introduced in gtk+ 3.20
[properties_dialog > dialog-vbox4 > vbox7 > table5 > file_perm_group_w_check]
	Property 'Focus on click' of object class 'Widget' was introduced in gtk+ 3.20
[properties_dialog > dialog-vbox4 > vbox7 > table5 > file_perm_group_r_check]
	Property 'Focus on click' of object class 'Widget' was introduced in gtk+ 3.20
[properties_dialog > dialog-vbox4 > vbox7 > table5 > file_perm_owner_x_check]
	Property 'Focus on click' of object class 'Widget' was introduced in gtk+ 3.20
[properties_dialog > dialog-vbox4 > vbox7 > table5 > file_perm_owner_w_check]
	Property 'Focus on click' of object class 'Widget' was introduced in gtk+ 3.20
[properties_dialog > dialog-vbox4 > vbox7 > table5 > file_perm_owner_r_check]
	Property 'Focus on click' of object class 'Widget' was introduced in gtk+ 3.20
[properties_dialog > dialog-vbox4 > vbox7 > table5 > label22]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table5 > label21]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table5 > label20]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table5 > label18]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table5 > label17]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table5 > label16]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table5 > label15]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > file_accessed_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > file_changed_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > file_modified_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > file_encoding_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > file_read_only_check]
	Property 'Focus on click' of object class 'Widget' was introduced in gtk+ 3.20
[properties_dialog > dialog-vbox4 > vbox7 > table3 > file_location_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > file_size_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > label14]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > label13]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > label12]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > label11]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > label10]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > label9]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > label7]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > file_type_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > table3 > label6]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[properties_dialog > dialog-vbox4 > vbox7 > hbox3 > file_name_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > vbox_project_dialog_editor > frame40_project > alignment48 > table18 > label240]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > vbox_project_dialog_editor > frame40_project > alignment48 > table18 > label241]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > table17 > label229]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > table17 > label228]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > table17 > label230]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > table17 > label26]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > table_project_dialog_project > label_project_dialog_filename]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > table_project_dialog_project > label_project_dialog_file_patterns]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > table_project_dialog_project > label_project_dialog_base_path]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > table_project_dialog_project > label_project_dialog_description]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > table_project_dialog_project > label_project_dialog_description]
	Property 'Y align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > table_project_dialog_project > label_project_dialog_name]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[project_dialog > project_dialog_vbox > project_notebook > table_project_dialog_project > label_project_dialog_filename0]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > frame_term > alignment1 > vbox_terminal > table_terminal1 > terminal_shell_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > frame_term > alignment1 > vbox_terminal > table_terminal1 > terminal_scrollback_lines_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > frame_term > alignment1 > vbox_terminal > table_terminal1 > terminal_bg_color_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > frame_term > alignment1 > vbox_terminal > table_terminal1 > terminal_fg_color_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > frame_term > alignment1 > vbox_terminal > table_terminal1 > terminal_font_label]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox49 > frame21 > alignment24 > vbox9 > table6 > label218]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox49 > frame21 > alignment24 > vbox9 > table6 > label217]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox49 > frame21 > alignment24 > vbox9 > table6 > label216]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox49 > frame21 > alignment24 > vbox9 > table6 > label131]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox49 > frame21 > alignment24 > vbox9 > table6 > label130]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox49 > frame21 > alignment24 > vbox9 > table6 > label123]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox49 > frame21 > alignment24 > vbox9 > table6 > label129]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox49 > frame21 > alignment24 > vbox9 > table6 > label126]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox23 > frame20 > alignment23 > vbox2 > table1 > label171]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox23 > frame20 > alignment23 > vbox2 > table1 > label117]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox23 > frame20 > alignment23 > vbox2 > table1 > label97]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox18 > frame17 > alignment20 > vbox37 > table10 > label208]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox18 > frame17 > alignment20 > vbox37 > table10 > label147]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox18 > frame37 > alignment40 > vbox43 > vbox45 > label_open_encoding]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox18 > frame37 > alignment40 > vbox43 > vbox44 > label153]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > vbox18 > frame6 > alignment7 > vbox8 > vbox38 > table15 > label210]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook4 > vbox24 > frame8 > alignment11 > table7 > label156]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook4 > vbox24 > frame8 > alignment11 > table7 > label134]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook4 > vbox24 > frame8 > alignment11 > table7 > label133]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook4 > vbox39 > frame18 > alignment21 > vbox19 > table14 > label250]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook4 > vbox39 > frame18 > alignment21 > vbox19 > table14 > label205]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook4 > vbox39 > frame18 > alignment21 > vbox19 > table14 > label173]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook4 > vbox39 > frame18 > alignment21 > vbox19 > table14 > label223]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook4 > vbox40 > frame27 > alignment30 > vbox25 > table13 > label200]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook4 > vbox40 > frame27 > alignment30 > vbox25 > table13 > label183]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook4 > vbox40 > frame27 > alignment30 > vbox25 > table13 > label222]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook4 > vbox40 > label247]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook6 > vbox54 > frame9 > alignment12 > table8 > label159]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook6 > vbox54 > frame9 > alignment12 > table8 > label161]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook6 > vbox54 > frame9 > alignment12 > table8 > label160]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook6 > vbox54 > frame29 > alignment32 > vbox26 > table21 > label150]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook6 > vbox14 > frame4 > alignment5 > table2 > label103]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook6 > vbox14 > frame4 > alignment5 > table2 > label101]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook6 > vbox14 > frame4 > alignment5 > table2 > label100]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook5 > vbox20 > frame25 > alignment28 > table11 > label235]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook5 > vbox20 > frame25 > alignment28 > table11 > label192]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
[prefs_dialog > dialog-vbox3 > notebook2 > notebook5 > vbox20 > frame25 > alignment28 > table11 > label191]
	Property 'X align' of object class 'Label' was introduced in gtk+ 3.16
```

Reply:
Geany has abandoned IRC after the shenanigans on freenode.

There is a #geany on matrix, although personally I don't use it much, but others do.

However if you want to discuss Geany stuff outside of a PR or issue Github discussions are enabled for Geany but not geany-plugins AFAIK.

Reply:
Set the supported version in Glade to 3.20.